test/python/fe_api/sdpa/test_sdpa_fwd_sm80.py

```
@@ -0,0 +1,177 @@
+
+
+def _bshd_randn(b, h, s, d, **kw):
+    """BHSD-logical tensor with the BSHD-physical stride order the SM80
+    adapters require."""
+    return torch.randn((b, s, h, d), **kw).permute(0, 2, 1, 3)
+
+
+def _ref_sdpa(q, k, v, *, is_causal, window_size, scale):
+    """Reference SDPA in fp32 with the same masking semantics the kernel
+    promises (causal / SWA window described by (left, right))."""
+    b, h_q, s_q, d_qk = q.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `b` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)